tests/common/snappi/common_helpers.py

```diff
@@ -359,11 +358,13 @@ def pfc_class_enable_vector(prio_list):
     return vector
 
 
-def get_wred_profiles(host_ans):
+def get_wred_profiles(host_ans, asic_value="None"):
     """
     Get all the WRED/ECN profiles of a SONiC switch
     Args:
         host_ans: Ansible host instance of the device
+        asic_value: asic value of the host
```

Comment:
Since you prefer to keep this parameter as `asic_value`, can you reference namespace in the description? The reason I prefer `asic_namespace` is because this is what it is. For a single ASIC system, we use the default namespace (i.e. None) and not the ASIC ID or value.

Reply:
Sure , this can done